src/Resources/translations/SonataAdminBundle.en.xliff

```diff
@@ -490,6 +490,14 @@
                 <source>toggle_navigation</source>
                 <target>Toggle Navigation</target>
             </trans-unit>
+            trans-unit id="label_type_starts_with">
```

Comment:
You made a typo here.

And I don't know the strategy of sonata to add new translation. What about other language ?

Reply:
I am not master of other languages. So no idea. :D

Reply:
Seems like, the key are added in every file, waiting for someone to translate them. This has the benefit to avoid conflict.

https://github.com/sonata-project/SonataAdminBundle/blob/3.x/src/Resources/translations/SonataAdminBundle.cs.xliff#L471

Can you do it ?

BTW, for the fr file it will be
```
<trans-unit id="label_type_starts_with">
    <source>label_type_starts_with</source>
    <target>commence par</target>
</trans-unit>
<trans-unit id="label_type_ends_with">
    <source>label_type_ends_with</source>
    <target>termine par</target>
</trans-unit>
```

Reply:
Okay seems a lot of changes now. Was thinking a simple PR. :). Let me try. Thanks

Reply:
Done. :)